### DISCUSSION TITLE
How does release `fix116` for 1.0.3 differ to its counterpart?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [bpozdena/OneDriveGUI#134] [Q&A] [answered]
How does

*	https://github.com/bpozdena/OneDriveGUI/releases/download/v1.0.3/OneDriveGUI-1.0.3_fix116-x86_64.AppImage differ to
*	https://github.com/bpozdena/OneDriveGUI/releases/download/v1.0.3/OneDriveGUI-1.0.3-x86_64.AppImage?

**Comment by bpozdena** [comment, marked as answer]
[OneDriveGUI-1.0.3_fix116-x86_64.AppImage](https://github.com/bpozdena/OneDriveGUI/releases/download/v1.0.3/OneDriveGUI-1.0.3_fix116-x86_64.AppImage) fixes crash reported in #116 . I only had to change one string to fix a problem one one specific distro, which was not worth releasing a new version. 

If the base 1.0.3 version is not crashing for you, just keep using that.

**Reply by roke-julian-lockhart** [reply]
**https://github.com/bpozdena/OneDriveGUI/discussions/134#discussioncomment-7603196**

> If the base 1.0.3 version is not crashing for you, just keep using that.

Thanks for the advice, @bpozdena.
